data/operators/office/government.json

```diff
@@ -195,15 +250,30 @@
       "displayName": "Department of Education (Philippines)",
       "id": "departmentofeducation-7987d4",
       "locationSet": {"include": ["ph"]},
+      "matchNames": ["deped", "deped city schools division", "deped school division"],
       "tags": {
         "government": "education",
         "office": "government",
         "operator": "Department of Education",
+        "operator:en": "Department of Education",
         "operator:short": "DepEd",
         "operator:tl": "Kagawaran ng Edukasyon",
         "operator:wikidata": "Q3550346"
       }
     },
+    {
+      "displayName": "Department of Foreign Affairs (Philippines)",
+      "locationSet": {"include": ["ph"]},
+      "matchNames": ["dfa"],
+      "tags": {
+        "office": "government",
```

Comment:
Looks like you missed a `government` tag. Is this missed because there can be different values?

Reply:
Can't think of best value. Maybe ministry, but since these are also passport offices, this can go into a better government= value, but can't think of the right value.